Snippets/ssf2.sqlpromptsnippet

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Because you have to take the extra step of selecting all and then running "Format SQL" from sqlprompt.  I actually replaced the default ssf with the snippet found in ssf2.  It doesn't block, you just have to enter the full ssf2.   Just didn't want to replace the original in case some didn't like it.  It really makes things faster when developing.